test/Broadway/EventSourcing/AbstractEventSourcingRepositoryTest.php

```diff
@@ -164,7 +164,7 @@ public function it_publishes_decorated_events()
         $repository = new EventSourcingRepository(
             $this->eventStore,
             $this->eventBus,
-            '\Broadway\EventSourcing\TestEventSourcedAggregate',
+            get_class($this->createAggregate()),
```

Comment:
I can, but my tests failed because of this... Not sure why this would
suddenly break though. Will look at it again..
On 26 Jun 2015 10:34 pm, "Alexander" <notifications@github.com> wrote:

> In test/Broadway/EventSourcing/AbstractEventSourcingRepositoryTest.php
> <https://github.com/qandidate-labs/broadway/pull/171#discussion_r33393258>
> :
>
> > @@ -164,7 +164,7 @@ public function it_publishes_decorated_events()
> >          $repository = new EventSourcingRepository(
> >              $this->eventStore,
> >              $this->eventBus,
> > -            '\Broadway\EventSourcing\TestEventSourcedAggregate',
> > +            get_class($this->createAggregate()),
>
> This change doesn't seem related. Can you make it in a separate PR?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/qandidate-labs/broadway/pull/171/files#r33393258>.
>